sql/pg_duckdb--0.3.0--0.4.0.sql

```diff
@@ -452,3 +452,31 @@ CREATE OPERATOR pg_catalog.!~~* (
     RIGHTARG = duckdb.unresolved_type,
     FUNCTION = duckdb_unresolved_type_operator
 );
+
+CREATE FUNCTION duckdb.is_motherduck_enabled()
+RETURNS bool
+SET search_path = pg_catalog, pg_temp
+LANGUAGE C AS 'MODULE_PATHNAME', 'pgduckdb_is_motherduck_enabled';
+REVOKE ALL ON FUNCTION duckdb.is_motherduck_enabled() FROM PUBLIC;
+
+CREATE FUNCTION pgduckdb_fdw_handler()
+RETURNS fdw_handler
+AS 'MODULE_PATHNAME', 'pgduckdb_fdw_handler'
+LANGUAGE C STRICT;
+
+CREATE FUNCTION pgduckdb_fdw_validator(
+    options text[],
+    catalog oid
+)
+RETURNS void
+AS 'MODULE_PATHNAME', 'pgduckdb_fdw_validator'
+LANGUAGE C STRICT PARALLEL SAFE;
+
+CREATE FOREIGN DATA WRAPPER pg_duckdb
```

Comment:
I think we should call this foreign data wrapper `duckdb`:
```suggestion
CREATE FOREIGN DATA WRAPPER duckdb
```
That matches nicely with our table access method which is also called `duckdb` and our schema etc.